### PR TITLE
Fix to issue #116

### PR DESCRIPTION
**Pull request by kaidaniel82**
## Related issue(s)
Fixes ...

## Pre-merge checklist
- [x] Passing tests LOCALLY
- [ ] New tests added (if applicable)

Please note that, in order to pass the tests, you'll need to set up your Tastytrade credentials as repository secrets on your local fork. Read more at CONTRIBUTING.md.

### CODE REVIEW

File: tastytrade/account.py [hunk 1]
```diff
@@ -295,7 +295,7 @@ class TradingStatus(TastytradeJsonDataclass):
     is_in_day_trade_equity_maintenance_call: bool
     is_in_margin_call: bool
     is_pattern_day_trader: bool
-    is_portfolio_margin_enabled: bool
+    is_portfolio_margin_enabled: Optional[bool] = None
```
Comment (Graeme22) [inline tastytrade/account.py:298]: Can you move this down to where the other Optional args are?

**Review by Graeme22 — APPROVED** [review]
Looks good, just one small thing.

**Comment by kaidaniel82** [comment]
Done

**Review by Graeme22 — APPROVED** [review]

**Comment by kaidaniel82** [comment]
What is the next step? I never did a pull request before? Ar you going to release this now?

**Pull request merged**